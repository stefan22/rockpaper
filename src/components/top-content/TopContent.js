import React, {Component} from 'react';
import MainMenu from './MainMenu';
import '../../scss/main-content.scss';

class TopContent extends Component {
  render() {
    return (
      <div className='top-section-wrapper'>
        <MainMenu />
      </div>
    );
  }
}

export default TopContent;
