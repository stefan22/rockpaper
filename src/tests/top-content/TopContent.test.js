import React from 'react';
import {shallow, mount, render} from 'enzyme';
import toJSON from 'enzyme-to-json';
import TopContent from '../../components/top-content/TopContent';

describe('TopContent component', () => {
  it('should render TopContent component correctly', () => {
    const wrapper = shallow(<TopContent />);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });


});
