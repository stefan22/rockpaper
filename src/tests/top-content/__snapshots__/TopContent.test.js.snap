// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TopContent component should render TopContent component correctly 1`] = `
<div
  className="top-section-wrapper"
>
  <MainMenu />
</div>
`;
